### PR TITLE
Add connection counts, most used bricks @cluster and system

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
Verified

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat @shtripat Please review

File: tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py [hunk 1]
```diff
@@ -160,6 +158,23 @@ def get_most_used_volumes(self, cluster_det):
...
+    def get_most_used_bricks(self, volumes_det):
+        brick_utilizations = []
+        for volume_id, volume_det in volumes_det.iteritems():
+            for brick_path, brick_det in volume_det.get(
+                'Bricks',
+                {}
+            ).iteritems():
+                brick_det['brick_path'] = brick_path
+                brick_det['vol_name'] = volume_det['name']
+                brick_utilizations.append(brick_det['utilization'])
```
Comment (r0h4n) [inline tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py]: what if brick_det['utilization'] is None

Comment (anmolbabu) [inline tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py]: Is that a possibility??

Comment (anmolbabu) [inline tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py]: Done

**Pull request merged**